### DISCUSSION TITLE
Dịch "helper"

### DISCUSSION THREAD

**Discussion by quangvnai** [d2l-ai/d2l-vi#2701] [Terminology]
Từ này xuất hiện cũng khá phổ biến trong cuốn sách này ví dụ như "helper function", và có lẽ mọi người dịch là hàm phụ trợ.

Mình google thì thấy từ helper function lại hay đi kèm từ hỗ trợ hơn. Mọi người hãy vote xem thống nhất dịch từ này là

1. **hàm phụ trợ**  👍 
2. **hàm hỗ trợ**  😄 

bằng cách thả icon nhé :))

cc: @minhduc0711 @cuongvng @lkhphuc @trGiang99  @rootonchair

**Comment by lkhphuc** [comment]
Mình không có strong opinion, chỉ là từ `phụ trợ` hay được dịch sát từ `auxiliary` nên mình vote `hỗ trợ`.

**Reply by rootonchair** [reply]
Em cũng đồng ý với ý kiến của @lkhphuc
